src/frameworks/foundation/ns_string.rs

```diff
@@ -435,6 +435,15 @@ pub const CLASSES: ClassExports = objc_classes! {
     new_string
 }
 
+- (id)lastPathComponent { // NSString*
+    let self_as_rust_string = &to_rust_string(env, this);
+    let path = GuestPath::new(self_as_rust_string);
+    let file_name = path.file_name().unwrap_or("");
```

Comment:
I just noticed that the path `"/"` is a special case, I guess that needs to be accounted for.